### ISSUE TITLE
Cannot override VCS_TYPE:  Cannot index object with number

### ISSUE THREAD

**Issue by tjerkw** [eddiewebb/circleci-queue#7]
The script fails for me, maybe circleci changed their api?

```
This build will block until all previous builds complete.
Max Queue Time: 5 minutes.
No Job variable, blocking on any runnin jobs for this project.
jq: error (at <stdin>:2): Cannot index object with number
API Call for existing jobs failed, failing this build.  Please check API token
Exited with code 1
```

**Comment by tjerkw** [comment]
Ah it was using github as VCS_TYPE. But it should have been bitbucket.

**Comment by tjerkw** [comment]
There is no way to override the VCS_TYPE its hardcoded to github. And the "repo_url" and the IF statement dont work.

I made it work by hacking around it (i didnt want to copy paste the scripts):

```
            # Ensure we use bitbucket in the api (hack
            sed -i -e 's/github/bitbucket/g' /home/circleci/circleci-queue/circle_api.sh
            # wait up to 5 minutes for previous builds
            CIRCLE_JOB= queueBuildUntilFrontOfLine 5
```

This should be fixed.

**Comment by tjerkw** [comment]
I submitted a PR with a solution

**Comment by eddiewebb** [comment]
Hi @tjerkw - it looks like that PR was opened against your own repo.  If you'd like to open it against this repo I would be happy to review and include it!

**Comment by tjerkw** [comment]
I submitted a PR now, but it has a conflict

**Comment by eddiewebb** [comment]
Fixed in #12
